Arch/README.md

```diff
@@ -0,0 +1,15 @@
+# Pros
+* **Customization capability**
+      After loading the ISO file for installation, the user can freely monitor and control various functional aspects, which is a fundamental distinguishing feature of Arch Linux.
+* **Flexible package management and integrity**
+      LINUX ARCH supports PACMAN management. This technology provides the ability to upgrade the entire system with a single pacman command - Syu. Loading of new versions of packages is performed by one click of pacman - S package-name.
+* **Arch Wiki**
+      your source for Arch Linux documentation on the web.
+* **Originality** 
+      Unlike other distributions such as Ubuntu, which is based on Debian, the arch Linux operating system was created from the ground up, independent of any Linux distribution.
+* **Flexibility**
+      Working with Arch Linux is very pleasant in addition to its complexities. it is a very flexible operating system that you are able to change it as you like, Also You are free to add the required features by choosing modules from the time of its installation.     
+# cons
+* **Difficult to install**
```

Comment:
Maybe let's not make these bold, and add a bit of detail. For example: 
- Can be difficult to install and set up for the average user
- Not very usable out of the box, and requires a lot of configuration